Examples/UIExplorer/ViewExample.js

```diff
@@ -188,5 +188,43 @@ exports.examples = [
         </View>
       );
     },
+  }, {
+    title: 'ZIndex',
+    render: function() {
+      const commonStyles = {
+        justifyContent: 'space-around',
+        width: 100,
+        height: 50,
+        marginTop: -10,
+      };
+      return (
+        <View>
+          <View style={[
+            commonStyles,
+            {marginTop: 0, backgroundColor: '#E57373', zIndex: 2}
+          ]}>
+            <Text>ZIndex 2</Text>
+          </View>
+          <View style={[
+            commonStyles,
+            {marginLeft: 50, backgroundColor: '#FFF176', zIndex: 1}
+          ]}>
+            <Text>ZIndex 1</Text>
+          </View>
+          <View style={[
+            commonStyles,
+            {marginLeft: 100, backgroundColor: '#81C784', zIndex: 0}
+          ]}>
+            <Text>ZIndex 0</Text>
+          </View>
+          <View style={[
+            commonStyles,
+            {marginLeft: 150, backgroundColor: '#64B5F6', zIndex: -1}
+          ]}>
+            <Text>ZIndex -1</Text>
```

Comment:
Maybe I'm missing something, but shouldn't a view with zIndex = -1 appear *underneath* a view view with zIndex = 0? In testViewExample_1@2x.png, it's on top.